build/release-notes.md

```diff
@@ -8,6 +8,127 @@ toc_max: 2
+
+## 0.10.1
+
+{% include release-date.html date="2023-01-27" %}
+
+> **Note**
+>
+> Buildx v0.10 enables support for a minimal [SLSA Provenance](https://slsa.dev/provenance/)
+> attestation, which requires support for [OCI-compliant](https://github.com/opencontainers/image-spec)
+> multi-platform images. This may introduce issues with registry and runtime support
+> (e.g. [Google Cloud Run and AWS Lambda](https://github.com/docker/buildx/issues/1533)).
+> You can optionally disable the default provenance attestation functionality
+> using `--provenance=false`.
+
+### Bug fixes and enhancements
+
+* Fix sending the correct origin URL as `vsc:source` metadata. {% include github_issue.md repo="docker/buildx" number="1548" %}
+* Fix possible panic from data-race. {% include github_issue.md repo="docker/buildx" number="1504" %}
+* Fix regression with `rm --all-inactive`. {% include github_issue.md repo="docker/buildx" number="1547" %}
+* Improve attestation access in `imagetools inspect` by lazily loading data. {% include github_issue.md repo="docker/buildx" number="1546" %}
+* Correctly mark capabilities request as internal. {% include github_issue.md repo="docker/buildx" number="1538" %}
+* Detect invalid attestation configuration. {% include github_issue.md repo="docker/buildx" number="1545" %}
+* Update containerd patches to fix possible push regression affecting
+  `imagetools` commands. {% include github_issue.md repo="docker/buildx" number="1559" %}
+
+## 0.10.0
+
+{% include release-date.html date="2023-01-10" %}
+
+> **Note**
+>
+> Buildx v0.10 enables support for a minimal [SLSA Provenance](https://slsa.dev/provenance/)
+> attestation, which requires support for [OCI-compliant](https://github.com/opencontainers/image-spec)
+> multi-platform images. This may introduce issues with registry and runtime support
+> (e.g. [Google Cloud Run and AWS Lambda](https://github.com/docker/buildx/issues/1533)).
+> You can optionally disable the default provenance attestation functionality
+> using `--provenance=false`.
+
+### New
+
+* Build command supports new flags `--attest` and shorthands `--sbom` and
+  `--provenance` for adding attestations for your current build.
+  `--attest type=sbom` or `--sbom=true` will generate SBOM (Software Bill of
+  Materials) attestation for your build result in SPDX format.
+  `--attest type=provenance` or `--provenance=true` will generate [SLSA provenance attestation](attestations/slsa-provenance.md)
+  for your build result with information about how the build was performed.
+  When creating OCI images, a minimal provenance attestation is included with
+  the image by default. This feature requires BuildKit v0.11.0+. Read more
+  about advanced [SBOM options in our dedicated docs](attestations/sbom.md). {% include github_issue.md repo="docker/buildx" number="1412" %}
+  {% include github_issue.md repo="docker/buildx" number="1475" %}
```

Comment:
I have no idea if I managed to format this correctly in the GitHub review GUI, but I tried breaking this into some sub-bullets for easier reading.

```suggestion
* The `buildx build` command supports a new `--attest` flag, along with shorthands `--sbom` and
  `--provenance`, for adding attestations for your current build. {% include github_issue.md repo="docker/buildx" number="1412" %}
  {% include github_issue.md repo="docker/buildx" number="1475" %}
  * `--attest type=sbom` or `--sbom=true` adds [SBOM attestations](attestations/sbom.md).
  * `--attest type=provenance` or `--provenance=true` adds [SLSA provenance attestations](attestations/slsa-provenance.md).
  * When creating OCI images, a minimal provenance attestation is included with
  the image by default.
```